README.md

```diff
@@ -6,27 +6,22 @@
 
 This repository contains the source content and code for [Kong](https://github.com/Kong/kong)'s documentation website. It's built using [Jekyll](https://jekyllrb.com/) and deployed with [Netlify](https://www.netlify.com/).
 
-Here are some things to know before you get started:
+## Getting started
+
+Here are some things you should know before getting started:
 
 * **We're beginner-friendly**. Whether you're a Technical Writer learning about docs-as-code or an engineer practicing your documentation skills, we welcome your involvement. If you'd like to contribute and don't have something in mind already, head on over to [Issues](https://github.com/Kong/docs.konghq.com/issues). We've added `good first issue` labels on beginner-friendly issues.
 
 * **We need more help in some areas**. We'd especially love some help with [plugin](https://github.com/Kong/docs.konghq.com/tree/main/app/_hub) documentation.
 
-* **Some of our docs are generated**.
-    * [Admin API](https://docs.konghq.com/gateway/latest/admin-api/)
-    * [CLI reference](https://docs.konghq.com/gateway/latest/reference/cli/)
-    * [OSS upgrade guide](https://docs.konghq.com/gateway/latest/install-and-run/upgrading-oss/)
-    * [PDK reference](https://docs.konghq.com/gateway/latest/pdk)
-
-All pull requests for these docs should be opened in the [Kong/kong](https://github.com/Kong/kong) repository. Fork the repository and submit PRs from your fork.
+* **Community is a priority for us**. Before submitting an issue or pull request, make sure to review our [Contributing Guide](https://docs.konghq.com/contributing/). 
 
-For [Gateway Enterprise configuration reference](https://docs.konghq.com/gateway/latest/reference/configuration), open an issue on this repo and we'll update the docs.
+* **Some of our docs are [generated](docs/autodocs.md)**.
 
-* **Community is a priority for us**. Before submitting an issue or pull request, make sure to review our [Contributing Guide](https://docs.konghq.com/contributing/) and our [documentation templates](https://github.com/Kong/docs.konghq.com/tree/main/docs/templates).
+* **We are currently accepting plugin submissions** to our plugin hub from trusted technical partners, on a limited basis. For more information, see the [Kong Partners page](https://konghq.com/partners/).
 
-* We are currently accepting plugin submissions to our plugin hub from trusted technical partners, on a limited basis. For more information, see the [Kong Partners page](https://konghq.com/partners/).
 
-## Run Locally
+### Running Locally
 
 For anything other than minor changes, [clone the repository onto your local machine and build locally](docs/platform-install.md). Once you've installed all of the tools required, you can use our `Makefile` to build the docs:
```

Comment:
Should we recommend this or using codespaces? Like if someone is just drafting something and wants our eyes on it and it's just text changes, I think codespaces would be okay since they don't want/need to preview it locally.

Reply:
I've tried to get code spaces to work but it always fails for me. I think it would be good though to have a sentence that says "we recommend using code spaces for small changes"

Reply:
Curious, what fails for you? I've never had issues, but I might be using it differently. I do think it's worth adding the two simple edit options for small changes:
* Click "edit this page" on any page to edit it directly in the GitHub UI
* Use codespaces (include a link to the GH codespaces doc to show them how it works)